src/content/docs/en/pages/products/edge-firewall/rules-engine.mdx

```diff
@@ -21,7 +21,9 @@ You can use the Rules Engine for Edge Firewall to configure all operational aspe
 
 An Edge Firewall Rule Set can have as many rules as needed. These rules are also interchangeable, as you can share the same rule for all other Edge Firewall configurations.
 
-> Learn more about [Edge Firewall](/en/documentation/products/edge-firewall/) and its modules.
+:::note
+Learn more about [Edge Firewall](/en/documentation/products/edge-firewall/) and its modules.
+:::
```

Comment:
```suggestion
import Button from '~/components/Button.astro';

<Button href="/en/documentation/products/edge-firewall/" text="go to Edge Firewall reference" variant="secondary"> 

</Button>
```

Reply:
In these cases, we're using the button component. You need to import it once, and you can add one or more buttons.